### PR TITLE
st2 [1888][IMP] account_view_adj_asx: show invoices in payment form

### PR DESCRIPTION
**Pull request by TomohiroAbe-Quartile**
[1888](https://www.quartile.co/web#id=1888&action=771&model=project.task&view_type=form&menu_id=505)

### CODE REVIEW

File: account_view_adj_asx/views/account_payment_view.xml [hunk 1]
```diff
@@ -8,6 +8,51 @@
...
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//sheet" position="inside">
+                <group string="Invoices">
+                    <field name="invoice_ids" nolabel="1">
+                        <tree string="Invoice" create="false">
+                            <field
+                                name="partner_id"
+                                groups="base.group_user"
```
Comment (yostashiro) [inline account_view_adj_asx/views/account_payment_view.xml]:
```suggestion
```

File: account_view_adj_asx/views/account_payment_view.xml [hunk 2]
```diff
@@ -8,6 +8,51 @@
...
+                                string="Partner"
+                            />
+                            <field name="date_invoice" />
+                            <field name="number" />
+                            <field name="user_id" />
```
Comment (yostashiro) [inline account_view_adj_asx/views/account_payment_view.xml]:
```suggestion
                            <field name="user_id" string="In Charge" />
```

File: account_view_adj_asx/views/account_payment_view.xml [hunk 3]
```diff
@@ -8,6 +8,51 @@
...
+    </record>
+    <record id="view_account_payment_form" model="ir.ui.view">
+        <field name="name">account.payment.form</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//sheet" position="inside">
+                <group string="Invoices">
+                    <field name="invoice_ids" nolabel="1">
+                        <tree string="Invoice" create="false">
```
Comment (yostashiro) [inline account_view_adj_asx/views/account_payment_view.xml]:
```suggestion
                        <tree string="Invoices">
```

File: account_view_adj_asx/README.rst [hunk 4]
```diff
@@ -13,15 +13,15 @@ Accounting Views Adjustments
 .. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
-.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fasx--custom-lightgray.png?logo=github
-    :target: https://github.com/qrtl/asx-custom/tree/12.0/account_view_adj_asx
+.. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Fasx--custom-lightgray.png?logo=github
+    :target: https://github.com/qrtl/asx-custom/tree/10.0/account_view_adj_asx
```
Comment (yostashiro) [inline account_view_adj_asx/README.rst]:
```suggestion
    :target: https://github.com/qrtl/asx-custom/tree/12.0/account_view_adj_asx
```

File: account_view_adj_asx/views/account_payment_view.xml [hunk 5]
```diff
@@ -8,6 +8,51 @@
...
+                            <field
+                                name="partner_id"
+                                groups="base.group_user"
+                                string="Partner"
+                            />
+                            <field name="date_invoice" />
+                            <field name="number" />
```
Comment (yostashiro) [inline account_view_adj_asx/views/account_payment_view.xml:27]:
```suggestion
                            <field name="number" />
                            <field name="name" />
```

File: account_view_adj_asx/readme/DESCRIPTION.rst [hunk 6]
```diff
@@ -0,0 +1,3 @@
+This module does the following:
+
+- Adds treeview in account_invoice and account_payment view.
```
Comment (yostashiro) [inline account_view_adj_asx/readme/DESCRIPTION.rst]:
```suggestion
- Adjusts payment form and tree views to show related invoices.
```

**Review by yostashiro — CHANGES_REQUESTED** [review]

File: account_view_adj_asx/views/account_payment_view.xml [hunk 7]
```diff
@@ -8,6 +8,51 @@
...
+                                name="amount_total_signed"
+                                string="Total"
+                                sum="Total"
+                            />
+                            <field
+                                name="residual_signed"
+                                string="Amount Due"
+                                sum="Amount Due"
+                            />
+                            <field name="state" />
```
Comment (yostashiro) [inline account_view_adj_asx/views/account_payment_view.xml:48]:
```suggestion
                            <field name="state" />
                            <field name="currency_id" invisible="1" />
```

File: account_view_adj_asx/views/account_payment_view.xml (hunk 5 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/views/account_payment_view.xml:27]: I added. Thank you.

File: account_view_adj_asx/views/account_payment_view.xml (hunk 7 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/views/account_payment_view.xml:48]: I added. Thank you.

File: account_view_adj_asx/views/account_payment_view.xml (hunk 1 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/views/account_payment_view.xml]: Deleted it.
Thank you.

File: account_view_adj_asx/README.rst (hunk 4 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/README.rst]: Thank you, I re-generate readme.

File: account_view_adj_asx/views/account_payment_view.xml (hunk 5 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/views/account_payment_view.xml:27]: Thank you, I added.

File: account_view_adj_asx/readme/DESCRIPTION.rst (hunk 6 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/readme/DESCRIPTION.rst]: Thank you, I edited and re-generated Readme.

File: account_view_adj_asx/views/account_payment_view.xml (hunk 7 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/views/account_payment_view.xml:48]: Thank you, I added.

File: account_view_adj_asx/views/account_payment_view.xml (hunk 2 above)
Comment (TomohiroAbe-Quartile) [reply account_view_adj_asx/views/account_payment_view.xml]: Thank you, I added.

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Pull request merged**